internal/store/postgres/alerts.go

```diff
@@ -99,7 +108,13 @@ func (r AlertRepository) List(ctx context.Context, flt alert.Filter) ([]alert.Al
 		return nil, err
 	}
 
-	rows, err := r.client.QueryxContext(ctx, query, args...)
+	// Instrumentation attributes
+	attrs := []attribute.KeyValue{
+		attribute.String("db.method", "Select *"),
```

Comment:
should be `db.operation` instead of `db.method`, please refer to the doc https://opentelemetry.io/docs/specs/semconv/database/sql/

Reply:
there is already an label `db.operation` which is giving information about the transaction beginning, execution or query.  
That is the reason for using the different label `db.method`.